subprojects/core/src/integTest/groovy/org/gradle/groovy/scripts/GroovyImportsIntegrationTest.groovy

```diff
@@ -39,4 +40,19 @@ import org.apache.commons.math3.util.FastMath
         succeeds("help")
     }
 
+    @IgnoreRest
```

Comment:
Is `@IgnoreRest` here for prototyping reasons?

Reply:
Yeah, I removed it now.